### PR TITLE
Add IPTV Stream Checker

### PR DESCRIPTION
**Pull request by NewsGuyTor**
IPTV Stream Checker is a command-line tool designed to check the status of channels in an IPTV M3U8 playlist; it verifies if the streams are alive, captures screenshots, provides detailed information about video and audio streams, and identifies any potential issues like low framerates or mislabeled channels.

### CODE REVIEW

**Review by PopeyeTheSai10r — DISMISSED** [review]

**Comment by PopeyeTheSai10r** [comment]
LGTM.

@NewsGuyTor you try making the description shorter.

> IPTV Stream Checker is a command-line tool that checks channel status in an IPTV M3U8 playlist by verifying stream availability, capturing screenshots, providing video and audio details, and detecting issues like low framerates or mislabeled channels.

File: README.md [hunk 1]
```diff
@@ -213,6 +213,7 @@ Useful tools for working with IPTV.
 - [RockMyM3u](https://rockmym3u.com/) - M3U & Xtream Editor, create playlists and more online.
 - [m3u.in](https://m3u.in/) - Create, edit, sort channels/groups in m3u playlists online, merge two or more playlists into one, automatic updates, access logs, short links and more. (blocked inside the Russian Federation for now)
 - [IPTV Tools](http://www.iptvtools.net/) - A website that provides the necessary tools for IPTV, such as a link checking tool.
+- [IPTV Stream Checker](https://github.com/NewsGuyTor/IPTVChecker) - IPTV Stream Checker is a command-line tool designed to check the status of channels in an IPTV M3U8 playlist; it verifies if the streams are alive, captures screenshots, provides detailed information about video and audio streams, and identifies any potential issues like low framerates or mislabeled channels.
```
Comment (BellezaEmporium) [inline README.md:216]:
```suggestion
- [IPTV Stream Checker](https://github.com/NewsGuyTor/IPTVChecker) - Check the availability of the tested streams and pinpoints labeling anomalies (Python is required)
```

**Review by freearhey — APPROVED** [review]

**Review by NewsGuyTor — COMMENTED** [review]
`Command-line tool designed to check the status and capture screenshots of channels in an IPTV M3U8 playlist.` is better I think, the screenshot functionality is pretty fundamental.

**Comment by freearhey** [comment]
@NewsGuyTor Fixed.

**Review by freearhey — APPROVED** [review]

**Review by Alstruit — APPROVED** [review]

**Pull request merged**